packages/react-emotion/types/index.d.ts

```diff
@@ -1,120 +1,21 @@
 // TypeScript Version: 2.3
-// tslint:disable-next-line:no-implicit-dependencies
-import { StatelessComponent, ComponentClass, CSSProperties } from 'react';
-import { Interpolation as EmotionInterpolation } from 'emotion';
+import {
+  CreateStyled,
+  Interpolation,
+  StyledComponent,
+  StyledOptions,
+  Themed,
+} from 'create-emotion-styled';
+
+export {
+  CreateStyled,
+  Interpolation,
+  StyledComponent,
+  StyledOptions,
+  Themed,
+} from 'create-emotion-styled';
```

Comment:
Was importing everything twice intentional?

Reply:
@mitchellhamilton Oh... no, it's my mistake. Thank you for pointing this!